plugins/mem0-ops/scripts/_api.py

```
@@ -0,0 +1,121 @@
+        try:
+            data = json.dumps(body).encode() if body is not None else None
+            req = urllib.request.Request(
+                url,
+                data=data,
+                method=method,
+                headers={
+                    "Authorization": f"Token {api_key()}",
+                    "Content-Type": "application/json",
+                },
+            )
+            with urllib.request.urlopen(req, timeout=30) as r:
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)